tests/test_isrTask.py

```diff
@@ -137,7 +137,7 @@ def test_updateVariance(self):
         statAfter = computeImageMedianAndStd(self.inputExp.variance[self.amp.getBBox()])
         self.assertGreater(statAfter[0], statBefore[0])
         self.assertFloatsAlmostEqual(statBefore[0], 0.0, atol=1e-2)
-        self.assertFloatsAlmostEqual(statAfter[0], 8170.0195, atol=1e-2)
+        self.assertFloatsAlmostEqual(statAfter[0], 8171.2099, atol=1e-2)
```

Comment:
Why are any numbers in `test_isrTask` changing?  Did we touch this at all with these changes?  If so we did something wrong.